lib/spack/docs/module_file_support.rst

```diff
@@ -294,23 +296,24 @@ For example, consider the following on one system:
     # antlr@2.7.7%gcc@5.3.0~csharp+cxx~java~python arch=linux-SuSE11-x86_64
     module load linux-SuSE11-x86_64/antlr-2.7.7-gcc-5.3.0-bdpl46y
 
------------------------------
+----------------------------
 Auto-generating Module Files
------------------------------
+----------------------------
 
 Module files are generated by post-install hooks after the successful
-installation of a package. They are placed in the following directories
-under the Spack root:
+installation of a package. The following table summarizes the essential
+information associated with the different file formats (row headings)
```

Comment:
I've never heard that rows have "headings", it took me a bit to parse this.  I think you could just drop the parenthetical comment and it would read easily and clearly.